apps/frontend/public/locales/override/translation.json

```diff
@@ -9,5 +9,6 @@
   "FAP Review Template": "FAP Review Template",
   "FAP Review template": "FAP Review template",
   "Start of Fap review": "Start of Fap review",
-  "End of Fap review": "End of Fap review"
+  "End of Fap review": "End of Fap review",
+  "Instrument Contact": "Area Contact"
```

Comment:
Should this file still have "Instrument Contact" as the value? I think these are the default values, and get replaced by the values from the file in our config repo. I'm not sure ESS & ELI would want to use "Area Contact" as well